src/backend/backend.py

```diff
@@ -39,14 +41,16 @@ def __init__(self):
 
     @staticmethod
     def _init_websocket():
-        while True:
+        app_started = False
+        while not app_started:
```

Comment:
I think `while` loops should make it clear from line one what is the exit condition. So I try to convert the "middle of the loop break" to a named stop condition I use in the top of the loop